src/cmd.js

```diff
@@ -434,6 +434,21 @@ const none = Object.freeze({
   simulate: () => null,
 });
 
+export function flattenCmd(cmd) {
+  if (
+    cmd.type === cmdTypes.MAP ||
+    cmd.type === cmdTypes.SET_TIMEOUT ||
+    cmd.type === cmdTypes.SET_INTERVAL
+  ) {
+    return flattenCmd(cmd.nestedCmd);
```

Comment:
we lose some info here. A map cmd is more than just the nested Cmd. it's also the tagger function. Is there any way we can preserve that in this?

Reply:
For our tests we haven't ever really felt the need to consider the tagger, but I do see how that can be a relevant detail. I also suppose even `Cmd.timeout` and `Cmd.interval` could be.

[Here was the solution I landed on, I think it makes the function very agnostic to what level of detail people are seeking to test against](https://github.com/redux-loop/redux-loop/pull/263/files#diff-5a04a1234d0d58e4a4a0221acde6f8f8909110c0100ceefd6b75621c1e6e6a6cR67-R82)